gabbi/suitemaker.py

```diff
@@ -217,12 +217,16 @@ def test_suite_from_dict(loader, test_base_name, suite_dict, test_directory,
 
     # Merge global with per-suite defaults
     default_test_dict = copy.deepcopy(case.HTTPTestCase.base_test)
+    seen_keys = set()
     for handler in handlers:
         default_test_dict.update(handler.test_base)
+        if handler.response_handler:
+            key = 'response_%s' % handler.test_key_suffix
```

Comment:
since key is getting thrown away outside this look, constructing 'key' from 'response_%s' is probably not necessary (as it will be done for every suffix). Should be okay to use `handler.test_key_suffix` directly in the set.